assets/scss/_content.scss

```diff
@@ -22,7 +22,9 @@
 
   article {
     a:where(.external-link)::after {
-      content: "⬈"
+      @include fa-icon;
+      content: fa-content($fa-var-external-link);
+      padding-left: 0.5rem;
```

Comment:
I'm not totally sure how small you'd like it to be, but I tried using rem and my issue was that there's a different overall font size dependent on screen width and rem doesn't correctly resize for that.  As such, I'm changing everything to em so that the <body> conditionals are taken into account, but I'm open to comments that this is still too big/small.

Reply:
`rem` should have worked because it uses the root font-size as base and changes the value relative to it.
But, no problem. I'm merging your PR, and later I try to tweak a bit with these values.